manifests/server/firewall.pp

```diff
@@ -11,15 +11,15 @@
     firewall { "${http_port} accept - puppetdb":
       dport  => $http_port,
       proto  => 'tcp',
-      action => 'accept',
+      jump   => 'accept',
```

Comment:
May want to reduce the extra space, in case of lint failures.

Reply:
Fixed